README.md

```diff
@@ -25,10 +25,10 @@ Please note that the action executes Az CLI script in a docker container. This m
 
 ### Dependencies on other GitHub Actions
 * [Azure Login](https://github.com/Azure/login) – **Optional**  Login with your Azure credentials, required only for authentication via azure credentials. Authentication via connection strings or keys do not require this step.
-
-    **Note** Make sure to either use default value of `azcliversion` or version above 2.30.0 for all the workflows using `GitHub hosted agents`.
+    > [!NOTE] Make sure to either use default value of `azcliversion` or version above 2.30.0 for all the workflows using `GitHub hosted agents`.

```

Comment:
Consider adding a line break after `> [!NOTE]` so that it can be rendered like

![image](https://github.com/Azure/cli/assets/4003950/9760ebb3-1885-447f-84ba-253b918fa986)

Reply:
> If you use this action, make sure to either use the default value of `azcliversion` or `azcliversion >= 2.30.0` for all the workflows. Authentication via connection strings or keys do not require this step.